src/style.css

```
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  padding: 0 50px;
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+}
+
+.nav__item {
+  display: flex;
+  flex-direction: row;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
+
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav__link.is-active::after {
+  content: "";
+  bottom: 0;
+  left: 0;
+  right: 0;
+  position: absolute;
+
+  height: 4px;
+  background-color: #00acdc;
+  color: #00acdc;
+  border-radius: 2px;
+  transition: 0.3s;
```

👍 
```suggestion
  transition: color 0.3s;
```
It is always better to specify the transition property
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x